draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -3456,808 +3548,1280 @@ ObjectLists MAY combine regular objects and secondary ObjectLists in the same ob
+Host: dcdn.example
+Accept: */*
+Content-Type: application/cdni; ptype=ci-trigger.v2
+Content-Length: 527
+
+{
+  "action": "preposition",
+  "specs": [{
+    "trigger-subject": "content",
+    "generic-trigger-spec-type": "content-objectlist",
+    "generic-trigger-spec-value": {
+      "objects": [{
+        "href": "https://www.example.com/hls/09000b67/index.m3u8",
+        "type": "hls"
+      }]
+     }
+  }],
+  "extensions": [
+    "generic-trigger-extension-type": "execution-policy",
+    "generic-trigger-extension-value": {
+      "depends": [
+        "https://dcdn.example/cit/564cc45e-9099-4a37-b95e-60342f2647ba"
+      ]
+    }
+  ] 
+}
+
 
 RESPONSE:
-  HTTP/1.1 304 Not Modified
-  Content-Length: 0
-  Expires: Wed, 04 May 2016 08:49:11 GMT
-  Server: example-server/0.1
-  ETag: "4331492443626270781"
-  Cache-Control: max-age=60
-  Date: Wed, 04 May 2016 08:48:11 GMT
-  Content-Type: application/cdni; ptype=ci-trigger-collection
 
-REQUEST:
-
-  GET /triggers/0 HTTP/1.1
-  User-Agent: example-user-agent/0.1
-  Host: dcdn.example.com
-  Accept: */*
-  If-None-Match: "6990548174277557683"
+HTTP/1.1 201 Created
+Date: Sun, 27 Oct 2024 08:48:28 GMT
+Content-Length: 467
+Content-Type: application/cdni; ptype=ci-trigger.v2
+Location: https://dcdn.example/cit/f6dde35f-703f-49e9-bb80-4964dff3bca5
+Server: example-server/0.1
 
-RESPONSE:
+{
+  "ctime": 1730119708,
+  "etime": 1730119768,
+  "mtime": 1730119708,
+  "state": "pending",
+  "action": "preposition",
+  "specs": [{
+    "trigger-subject": "content",
+    "generic-trigger-spec-type": "content-objectlist",
+    "generic-trigger-spec-value": {
+      "objects": [{
+        "href": "https://www.example.com/hls/09000b67/index.m3u8",
+        "type": "hls"
+      }]
+     }
+  }],
+  "extensions": [
+    "generic-trigger-extension-type": "execution-policy",
+    "generic-trigger-extension-value": {
+      "depends": [
+        "https://dcdn.example/cit/564cc45e-9099-4a37-b95e-60342f2647ba"
+      ]
+    }
+  ] 
+}
 
-  HTTP/1.1 304 Not Modified
-  Content-Length: 0
-  Expires: Wed, 04 May 2016 08:49:10 GMT
-  Server: example-server/0.1
-  ETag: "6990548174277557683"
-  Cache-Control: max-age=60
-  Date: Wed, 04 May 2016 08:48:10 GMT
-  Content-Type: application/cdni; ptype=ci-trigger.v2
 ]]></artwork>
-   </figure>
-   <t>
-   When the trigger processing is complete, the contents of the
-   filtered collections will be updated along with their ETags. For
-   example, when the two example triggers are complete, the
-   collections of pending and complete triggers look as follows:</t>
+    </figure>
+    <t>
+    uCDN can also stagger long-running triggers to control processing order.
+    In the following example, uCDN creates a preposition triggers of normal priority, followed by a trigger of higher priority that 
+    dCDN should pick up for execution before the earlier trigger.
+    </t>
+
+    <figure>
+    <artwork><![CDATA[
 
-   <figure><artwork><![CDATA[
 REQUEST:
```

Comment:
In the interest in being concise, is the trigger with normal priority really necessary as part of the example?  Every example that has been POSTed thus far is a normal priority trigger.  Wouldn't it be sufficient to show how to POST a trigger with higher priority, and just say that this would be the highest priority job in the system?

Reply:
agreed, removed it and changed the commentary